app/util.py

```diff
@@ -39,6 +40,14 @@ def recordAPIStatus(apiName):
             if (r.status_code == 429):
                 e_code = r.json().get("code")
                 req = False
+        elif (apiName == "tripadvisor-mapper"):
+            ta_test_link = "http://api.tripadvisor.com/api/partner/2.0/location_mapper/37.774125,-122.422099"
```

Comment:
nit: would be great to use the constants/fns in the tripadvisor provider (feel free to make things public), e.g. so if the URL changes or we update to a new API version, this functionality comes for free.